db/migrate/20150714202351_create_users.rb

```diff
@@ -0,0 +1,10 @@
+class CreateUsers < ActiveRecord::Migration
+  def change
+    create_table :users do |t|
+      t.string :username
+      t.string :token
```

Comment:
Since we don't want these value to be `nil` you probably want to add `null: false`.